app/controllers/katello/api/v2/products_controller.rb

```diff
@@ -24,8 +25,8 @@ class Api::V2::ProductsController < Api::V2::ApiController
     api :GET, "/activation_keys/:activation_key_id/products", N_("List of subscription products in an activation key")
     api :GET, "/organizations/:organization_id/products", N_("List of products in an organization")
     api :GET, "/organizations/:organization_id/sync_plans/:sync_plan_id/available_products",
-      N_("List of available Products for sync plan")
-    api :GET, "sync_plans/:sync_plan_id/products", N_("List of Products for sync plan")
+      N_("List of available Products for sync plan"), :deprecated => true
+    api :GET, "/sync_plans/:sync_plan_id/products", N_("List of Products for sync plan")
```

Comment:
Should we switch the UI over to using this API method to retrieve products attached to a sync plan?